salt/pillar/elife-metrics.sls

```diff
@@ -39,3 +32,8 @@ elife_metrics:
         client_id: bar
         client_email: bup
         type: service_account
+
```

Comment:
The build finished correctly, but these pillars are not in https://github.com/elifesciences/builder-private/blob/master/pillar/elife-metrics.sls#L43
I think it's using this default: https://github.com/elifesciences/builder-private/blob/master/pillar/elife.sls#L234 which we want to change before merging.

Reply:
Yep:
```
[34.229.126.122] out:           ID: psql-app-db
[34.229.126.122] out:     Function: postgres_database.present
[34.229.126.122] out:         Name: appdb
[34.229.126.122] out:       Result: True
[34.229.126.122] out:      Comment: The database appdb has been created
[34.229.126.122] out:      Started: 09:36:32.351195
[34.229.126.122] out:     Duration: 315.888 ms
[34.229.126.122] out:      Changes:   
[34.229.126.122] out:               ----------
[34.229.126.122] out:               appdb:
[34.229.126.122] out:                   Present
```
https://alfred.elifesciences.org/blue/rest/organizations/jenkins/pipelines/pull-requests-projects/pipelines/elife-metrics-formula/branches/PR-4/runs/1/nodes/27/steps/30/log/?start=0